lib/v2/datasource/remote/model/transaction_model.dart

```diff
@@ -0,0 +1,47 @@
+class TransactionModel {
+  final String from;
+  final String to;
+  final String quantity;
+  final String memo;
+  final String timestamp;
+  final String transactionId;
+
+  String get symbol {
+    return quantity.split(" ")[1];
+  }
```

Comment:
I like arrow functions () => for small code getters, functios.

Reply:
yeah nicer ;)